src/terminologySectionPanel/codesSection/codesSubSection/codeSubSection/CodeSubSection.test.tsx

```diff
@@ -40,6 +48,11 @@ describe("CodeSub Section component", () => {
       "code-system-version-selector-input"
     );
     expect(codeSystemVersionSelectInput).toBeInTheDocument();
+    expect(codeSystemVersionSelectInput.value).toBe("2.0");
+    fireEvent.change(codeSystemSelectInput, {
+      target: { value: "1.0" },
+    });
+    expect(codeSystemVersionSelectInput.value).toBe("2.0");
```

Comment:
I found that if we have use attribute on HTML elements, our linter is happy when we declare the Element, Something like 

const codeSystemVersionSelectInput = getByTestId(
      "code-system-version-selector-input"
    ) as HTMLInputElement ;